src/structures/Guild.js

```diff
@@ -1174,6 +1192,21 @@ class Guild extends Base {
     return this.edit({ splash: await DataResolver.resolveImage(splash), reason });
   }
 
+  /**
+   * Sets a new guild discovery splash image.
+   * @param {Base64Resolvable|BufferResolvable} discoverySplash The new discovery splash image of the guild
+   * @param {string} [reason] Reason for changing the guild's discovery splash image
+   * @returns {Promise<Guild>}
+   * @example
+   * // Edit the guild discovery splash
+   * guild.setDiscoverySplash('./discoverysplash.png')
+   *  .then(updated => console.log('Updated the guild discovery splash'))
+   *  .catch(console.error);
```

Comment:
We are using two spaces of indentation for examples, however, there is only one space here:
```suggestion
   *   .then(updated => console.log('Updated the guild discovery splash'))
   *   .catch(console.error);
```

Reply:
All the other ones close to this use one space